tests/test_wasm_build.py

```
@@ -0,0 +1,289 @@
+    def test_style_css_exists(self):
+        """style.css should exist."""
+        style_css = WEB_DIR / "style.css"
+        assert style_css.exists(), "wasm/web/style.css not found"
+
+    def test_style_css_has_content(self):
+        """style.css should have CSS rules."""
+        style_css = WEB_DIR / "style.css"
+        content = style_css.read_text()
+
+        assert len(content) > 100, "style.css seems too short"
+        assert "{" in content and "}" in content, "No CSS rules found"
```

Ruff (0.14.10)

Assertion should be broken down into multiple parts

(PT018)